Dockerfile

```diff
@@ -12,7 +12,14 @@ LABEL maintainer=terraform-linters
 
 RUN apk add --no-cache ca-certificates
 
-COPY --from=builder /tflint/dist/tflint /usr/local/bin
+# copy tflint from builder
+COPY --from=builder /tflint/dist/tflint /usr/local/bin/tflint
 
-ENTRYPOINT ["tflint"]
+# set working directory
 WORKDIR /data
+
+# set the default entrypoint -- when this container is run, use this command
+ENTRYPOINT [ "tflint" ]
```

Comment:
Keep the formatting consistent with Docker's docs, meaning no array padding:

https://docs.docker.com/engine/reference/builder/